docs/docs/0.3.2/documentation/README.md

```diff
@@ -1841,8 +1847,40 @@ When `kuma-cp` starts up, by default it listens on a few ports:
 * `5682`: the HTTP server that provides the Envoy bootstrap configuration when the data-plane starts up.
 * `5683`: the HTTP server that exposes Kuma UI.
 
-## Quickstart
+### kuma-dp ports
+
+When we start a data-plane via `kuma-dp` we expect all the inbound and outbound service traffic to go through it. The inbound and outbound ports are defined in the [dataplane specification](#dataplane-specification) when running in universal mode, while on Kubernetes the service traffic always runs on port `15001`.
+
+In addition to the service traffic ports, the data-plane automatically also opens the `envoy` [administration interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) service on the following ports:
+
+* On Kubernetes, by default on port `9901`.
+* On Universal, by default on the first available port greater or equal than `30001`.
+
+The Envoy administration interface can also be manually configured to listen on any arbitraty port by specifying the `--admin-port` argument when running `kuma-dp`.
+
+### Service Discovery
+
+Here we are going to be exploring the communication between `kuma-dp` and `kuma-cp`, and the communication between multiple `kuma-dp` to handle our service traffic.
+
+Every time a data-plane (served by `kuma-dp`) registers itself to Kuma, it initiates a gRPC streaming connection to Kuma (served by `kuma-cp`) in order to retrieve the latest configuration and share data to the control-plane. 
 
-The getting started for Kuma can be found in the [installation page](/install/0.3.2) where you can follow the instructions to get up and running with Kuma.
+::: tip
+The connection between the data-planes and the control-plane is not on the execution path of the service requests, which means that if the data-plane temporarily loses connection to the control-plane the service traffic won't be affected.
+:::
+
+While doing so, the data-planes also advertise the IP address of each service. The IP address is retrieved:
+
+* On Kubernetes by looking at the address of the `Pod`.
+* On Universal by looking at the inbound listeners that have been configured in the [`inbound` property](#dataplane-specification) of the data-plane specification.
+
+The IP address that's being advertised by every data-plane to the control-plane is also being used to route service traffic from one `kuma-dp` to another `kuma-dp`. This means that Kuma knows at any given time what are all the IP addresses associated to every replica of every service.
+
+Because Kuma by design already knows the address to every data-plane - and therefore to every service - it is not required to use Kuma with a third-party service discovery tool or dns resolver, because such tools would not give Kuma any additional information than the one it already knows.
```

Comment:
> because such tools would not give Kuma any additional information than the one it already knows

There is always a use case of proxying traffic to services outside of the mesh. In that case a third-party service discovery tool would enable load-balancing